### PR TITLE
[NU-1275] MacOS-related scripts improvements

### PR DESCRIPTION
**Pull request by coutoPL**
Test scenarios:
1. Docker streaming:
```bash
$ ./docker/streaming/start.sh && ./common/tests/testDockerStreaming.sh
# verify if tests go well and clean up
$ ./docker/streaming/clean.sh
```

2. k8s request-response:
```bash
$  .k3d/remove-k3d-cluster.sh && .k3d/create-k3d-cluster.sh
$ ./k8s-helm/additional/install-request-response.sh && ./k8s-helm/install-request-response.sh && ./common/tests/testK8sRequestResponse.sh
# verify if tests go well and clean up
$ ./k8s-helm/uninstall-request-response.sh && .k3d/remove-k3d-cluster.sh 
```

3. k8s streaming:

```bash
$  .k3d/remove-k3d-cluster.sh && .k3d/create-k3d-cluster.sh
$ ./k8s-helm/additional/install-streaming.sh && ./k8s-helm/install-streaming.sh && ./common/tests/testK8sStreaming.sh
# verify if tests go well and clean up
$ ./k8s-helm/uninstall-streaming.sh && .k3d/remove-k3d-cluster.sh 
```

### CODE REVIEW

File: docker/common/docker-compose.yml [hunk 1]
```diff
@@ -3,21 +3,22 @@ version: '3.8'
...
       JDK_JAVA_OPTIONS: -Xmx256M
+      JAVA_TOOL_OPTIONS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
       DEFAULT_SCENARIO_TYPE: ${DEFAULT_SCENARIO_TYPE-}
       INFLUXDB_URL: http://influxdb:8086
-      OPENAPI_SERVICE_URL: http://customerservice:5000
+      OPENAPI_SERVICE_URL: http://customerservice:5001
       AUTO_OFFSET_RESET: ${AUTO_OFFSET_RESET-latest}
       USAGE_REPORTS_FINGERPRINT: ${USAGE_REPORTS_FINGERPRINT-}
       USAGE_REPORTS_SOURCE: "quickstart-docker-compose"
+      NUSSKNACKER_LOG_LEVEL: INFO
```
Comment (arkadius) [inline docker/common/docker-compose.yml:19]: Why did you add it if it is set already to INFO?

File: common/scripts/continuouslySendTransactions.sh [hunk 2]
```diff
@@ -2,20 +2,23 @@
...
-  AMOUNT=$((1 + `random_number` % 30))
-  NOW=`date +%s%3N`
-  TIME=$((NOW - `random_number` % 20))
+  ID=$((1 + $(random_4digit_number) % 5))
+  AMOUNT=$((1 + $(random_4digit_number) % 30))
+  NOW="$(date +%s)$(random_3digit_number)"
+  TIME=$((NOW - $(random_4digit_number) % 20))
   echo "{ \"clientId\": \"$ID\", \"amount\": $AMOUNT, \"eventDate\": $TIME}" | ./sendToKafka.sh transactions
-  echo -n "."
+  printf "."
```
Comment (arkadius) [inline common/scripts/continuouslySendTransactions.sh:22]: What? We can't use echo? :crying_cat_face:

File: common/schemas/createSchemas.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ cd $resolved_dir
 
 function wrapWithSchemaRegistryMessage() {
     NAME=$1
-    echo "{\"schema\": \"$(cat $NAME.json | sed -z -e 's/\n/\\n/g' -e 's/"/\\"/g')\", \"schemaType\": \"JSON\"}"
+    echo "{\"schema\": \"$(cat $NAME.json | awk '{gsub(/\n/, "\\n"); gsub(/"/, "\\\""); print}')\", \"schemaType\": \"JSON\"}"
```
Comment (arkadius) [inline common/schemas/createSchemas.sh:13]: W can't use sed? :crying_cat_face:

File: common/customerservice/Dockerfile [hunk 4]
```diff
@@ -4,4 +4,4 @@ ADD requirements.txt .
 RUN pip install -r requirements.txt \
     && rm -rf /root/.cache
 COPY . .
-CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "5000"]
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "5001"]
```
Comment (arkadius) [inline common/customerservice/Dockerfile:7]: What is on 5000 on macos?

File: common/scripts/continuouslySendTransactions.sh (hunk 2 above)
Comment (coutoPL) [reply common/scripts/continuouslySendTransactions.sh:22]: nope. No `-n` on macos: https://apple.stackexchange.com/questions/173836/why-echo-n-doesnt-work-in-this-script-on-mac-terminal

File: common/schemas/createSchemas.sh (hunk 3 above)
Comment (coutoPL) [reply common/schemas/createSchemas.sh:13]: nope. No `-z` option: https://ss64.com/osx/sed.html

File: common/customerservice/Dockerfile (hunk 4 above)
Comment (coutoPL) [reply common/customerservice/Dockerfile:7]: by default it's used starting from MacOS Monterey: 
https://medium.com/pythonistas/port-5000-already-in-use-macos-monterey-issue-d86b02edd36c

**Review by arkadius — APPROVED** [review]
LGTM

**Review by Dzuming — APPROVED** [review]
LGTM
